tabletApp/features/core/src/commonMain/kotlin/band/effective/office/tablet/ui/theme/Theme.kt

```diff
@@ -79,7 +79,7 @@ fun AppTheme(
     useDarkTheme: Boolean = isSystemInDarkTheme(),
     content: @Composable () -> Unit
 ) {
-   val colors = if (!useDarkTheme) {
+   /*val colors = if (!useDarkTheme) {
```

Comment:
why comment select theme? to disable the dark theme, just pass true to the useDarkTheme